README.md

```diff
@@ -67,3 +67,8 @@ git commit [...] # to save it it
 `image`: used for sharing the URL in Facebook and LinkedIn. Best fit: 1600px x 900px. If not specified, the [ENGR logo](./static/images/social/social-preview.png) will be used;
 
 `twitterImage`: used for the twitter cards when sharing the url of this page (in X - former Twitter -, Telegram, Slack, Whatsapp); You can use the same image as `image` parameter. If not specified, the [ENGR logo](./static/images/social/social-preview.png) will be used;
+
+## Hints on how to write post
+
+they're included [in this article](./how-to-write-posts.md).
```

Comment:
_they're included [in this article](./how-to-write-posts.md)._ -> The hints are listed [in this article](./how-to-write-posts.md).